DER/DerivativesContracts/RightsAndWarrants.rdf

```diff
@@ -132,6 +204,78 @@
+						<owl:someValuesFrom rdf:resource="&fibo-sec-eq-eq;ConvertibleCommonShare"/>
+					</owl:Restriction>
+				</owl:someValuesFrom>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">common convertible share subscription right</rdfs:label>
+		<skos:definition xml:lang="en">subscription right entitling existing common convertible shareholders to subscribe to new securities at a price normally lower than the prevailing market price</skos:definition>
+		<fibo-fnd-utl-av:adaptedFrom xml:lang="en">ISO 10962, Securities and related financial instruments - Classification of financial instruments (CFI) code, Fourth Edition, October 2019</fibo-fnd-utl-av:adaptedFrom>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-der-drc-raw;CommonSharePurchaseRight">
+		<rdfs:subClassOf rdf:resource="&fibo-der-drc-raw;PurchaseRight"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-drc-bsc;hasUnderlier"/>
+				<owl:someValuesFrom>
+					<owl:Restriction>
+						<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;hasIdentity"/>
+						<owl:someValuesFrom rdf:resource="&fibo-sec-eq-eq;CommonShare"/>
+					</owl:Restriction>
+				</owl:someValuesFrom>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">common share purchase right</rdfs:label>
+		<skos:definition xml:lang="en">purchase right that gives a prospective acquiree&apos;s common shareholders the right to buy common shares of the firm or common shares of anyone who acquires the firm at a deep discount to their fair market value</skos:definition>
+		<fibo-fnd-utl-av:adaptedFrom xml:lang="en">ISO 10962, Securities and related financial instruments - Classification of financial instruments (CFI) code, Fourth Edition, October 2019</fibo-fnd-utl-av:adaptedFrom>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-der-drc-raw;CommonShareSubscriptionRight">
+		<rdfs:subClassOf rdf:resource="&fibo-der-drc-raw;SubscriptionRight"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-drc-bsc;hasUnderlier"/>
+				<owl:someValuesFrom>
+					<owl:Restriction>
+						<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;hasIdentity"/>
+						<owl:someValuesFrom rdf:resource="&fibo-sec-eq-eq;CommonShare"/>
+					</owl:Restriction>
+				</owl:someValuesFrom>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">common share subscription right</rdfs:label>
+		<skos:definition xml:lang="en">subscription right entitling existing common shareholders to subscribe to new securities at a price normally lower than the prevailing market price</skos:definition>
+		<fibo-fnd-utl-av:adaptedFrom xml:lang="en">ISO 10962, Securities and related financial instruments - Classification of financial instruments (CFI) code, Fourth Edition, October 2019</fibo-fnd-utl-av:adaptedFrom>
+	</owl:Class>
+	
 	<owl:Class rdf:about="&fibo-der-drc-raw;CompanyWarrant">
```

Comment:
Insufficient: we want the restriction to say that it's issued by the SAME company that issued the equity linked to the EquityWarrant.